tests/v2.test.js

```diff
@@ -0,0 +1,443 @@
+import expect from 'expect';
```

Comment:
`expect` is automatically global within a Jest test suite so you don't need to import it.

Reply:
Think that was to make my IDE happier!